shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md

```diff
@@ -13,17 +13,16 @@ TODO: परिष्करणीयम्
 
 प्रपन्न धर्मसार समुच्चयम् 
 
-इदंकिलान्वर्थ नामधेयं प्रबन्धरत्नं श्रुत पूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थ पर्यवेक्षण समुत्कण्ठानामास्तिकशिरोमणीनाम्। तदिदमिदानीं दैवादधिगतमस्माभिः। 
+इदं किलान्वर्थ नामधेयं प्रबन्धरत्नं श्रुतपूर्वमप्यदृष्टचरमिति नपुनरावेदनीयमुत्तमग्रन्थ पर्यवेक्षण समुत्कण्ठानामास्तिकशिरोमणीनाम्। तदिदमिदानीं दैवादधिगतमस्माभिः। 
 
 * 
 
-अयङ्किल परिभाषा अभिगमन मुपादान मिळ्या स्वाध्या य योगपरिच्छेद रूपेण पोढाविभक्तः। एकैक परिच्छेदे ष्वपि पाञ्चकालिन्धर्मानुष्ठानपयोग्यंशा बहवः प्रतिपाद्य स्ते। पञ्चशाल व्यवस्था, आन्ह्मिक प्रक्रियाः, तदुपयुक्तमन्तादि, तेषा मर्थनिरूपणम्, तत्तत्प्रमेयानुगुण प्रमाणप्रदर्शनम्,। प्रयोगा श्चेत्यादयो बहवो विषमा स्सारतमाः प्रतिपा द्यन्ते। किञ्चपारमैशास्त्य निष्टानां श्रीपाञ्चरात्रादि भग वच्छास्त्र शरणानां पाञ्चशालिक धर्मानुष्टान तत्पराणा मव श्यजिज्ञास्या स्सारतमाश्च विषयाः एकैक परिच्छेदेषु परळ्ळत माविष्क्रियन्ते। 
+अयङ्किल परिभाषा अभिगमनमुपादान मिळ्या स्वाध्याययोगपरिच्छेद रूपेण षोढाविभक्तः। एकैक परिच्छेदेष्वपि पाञ्चकालिन्धर्मानुष्ठानूपयोग्यंशा बहवः प्रतिपाद्यन्ते। पञ्चकालव्यवस्था, आह्निकप्रक्रियाः, तदुपयुक्तमन्त्राः, तेषामर्थनिरूपणम्, तत्तत्प्रमेयानुगुण प्रमाणप्रदर्शनम्, प्रयोगाश्चेत्यादयो बहवो विषयास्सारतमाः प्रतिपाद्यन्ते। किञ्चपारमैकान्त्यनिष्टानां श्रीपाञ्चरात्रादि भगवच्छास्त्र शरणानां पाञ्चकालिक धर्मानुष्टान तत्पराणामवश्यजिज्ञास्यास्सारतमाश्च विषयाः एकैक परिच्छेदेषु परश्शतमाविष्क्रियन्ते। 
 
-ईदृश ग्रस्थरत्न मुद्रणाय श्रीश्री ब्रह्मतस्त्र परकाल महादेशि केन्द्रस्थाना दनुगृहीत मेकं पुस्तकम्, तथा मैनू र् राजकीय कोशागारा देकम्, अन्य चीरङ्गपट्टणा वासि श्रीनि वासरङ्गभट्टा रै ्यरधिगतम्, रामनाधपुरावासि श्रीरामक्रतु नाथवम्भ्यै श्शठकोपाचार्यै स्सम्प्राप्तगुपरम्, एतचीळोळ चतुष्टयावलोकन पूर्वकं यथामति संशोधित०चदानेन, 
+ईदृश ग्रस्थरत्नमुद्रणाय श्रीश्रीब्रह्मतन्त्र परकालमहादेशिकेन्द्रस्थानादनुगृहीतमेकं पुस्तकं, तथा मैसूर् राजकीयकोशागारादेकम्, अन्यच्छ्रीरङ्गपट्टणावासि श्रीनिवासरङ्गभट्टार्यैरधिगतम्, रामनाधपुरावासि श्रीरामक्रतुनाथवंश्यैश्शठकोपाचार्यैस्सम्प्राप्तनुपरम्, एतच्छ्रीकोशचतुष्टयावलोकनपूर्वकं यथामति संशोधितञ्दासेन। 
 
-( क ) 
-
-अस्य ग्रस्थस्य प्रणेतारो श्री वङ्गिवं शेश्वरान्वय कलशप योराशि राशाशशाङ्काः श्रीपै कुण्णदीक्षिताचार्याः, अत एवास्य वैकुण वीक्षितीय मितिच प्रसिद्धिः, एते वैदिक सार्वभौम काला दर्वाचीना एन, तथापि शीष्टानुष्ठानयोग्यं बहवः प्रतिपा द्यन्त इति मैसूर् राजकीय मुजराय् सूपरिण्टॆण्डॆण्ट् प्र भूणा माज्ञाधार केण दा सेन मुद्रापि लेस्मिन् ग्रन्थे अक्षर विन्या सर्खालित्यादिदोष सद्भावेसि "वरङ्क णे नियच्छति "ति न्यायेन निर्मि तृरा गुणग्राहिण स्तत्र भवन्तो महा स्सहृदयास्साड रं क्षन्तु महळा स्त्रीति सविनयं साखुलिबन्धं प्रार्थये। 
+***marker***
```

Comment:
hmm - just replace with underscores (we're using markdown) -

____________________________

Reply:
Done

Reply:
Remove the text "marker" as well ...